sanghyeok/LV_1/2016년.js

```diff
@@ -0,0 +1,31 @@
+function solution(a, b) {
+    //윤년 : 2월이 29일까지  
+    let month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+    let week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
+    let sum = b;
+    for (let i = 0; i < a-1; i++) sum += month[i];
+  
+    //console.log(sum)  
+    switch(sum % 7) {
+     case 0 :
+           return week[4];  
+            break;
+     case 1 :
+           return week[5];
+            break;
+     case 2 :
+           return week[6];
+            break;
+     case 3 :
+           return week[0];
+            break;
+     case 4 :
+           return week[1];
+            break;
+     case 5 :
+           return week[2];
+            break;
+     case 6 :
+           return week[3];
+    }
+}
```

Comment:
```suggestion
function solution(a, b) {
    //윤년 : 2월이 29일까지  
    let month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    let week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
    let sum = b;
    for (let i = 0; i < a-1; i++) sum += month[i];
  
    //console.log(sum)  
    switch(sum % 7) {
     case 0 :
           return week[4];  
            break;
     case 1 :
           return week[5];
            break;
     case 2 :
           return week[6];
            break;
     case 3 :
           return week[0];
            break;
     case 4 :
           return week[1];
            break;
     case 5 :
           return week[2];
            break;
     case 6 :
           return week[3];
    }
}
```
switch문을 활용한 풀이법 잘 봤습니다!
Date.prototype.getDay() 매서드를 활용한 풀이도 알고가시면 좋을 것 같아요~!

Reply:
getDay 메서드가 있다는 생각을 못하고 풀었네요 ㅠ 희지님 코드 보면서 이해할 수 있었습니다, 코드 피드백 감사합니다!